### PR TITLE
feat: support record comment

### LINKED ISSUE #833: Comments field type
**Is your feature request related to a problem? Please describe.**

Data are social artifacts which often need discussion. Unfortunately there are no field, cell or table comments. 

**Describe the solution you'd like**

Here we focus on a comments field type. A record with this kind of type gets created an invisible junction table, in which comments with a predefined schema (ID, author, timestamps, text) are retained. There are special views for comments in the different places where they can appear.

This will esp. be a great addition to the expanded card view of a single record.

**Describe alternatives you've considered**

Adding comments can be improvised with referencing another table, but they would not receive the specific UI one would need to seamlessly use them.

### LINKED ISSUE #834: Cell comments
**Is your feature request related to a problem? Please describe.**

Data are social artifacts which often need discussion. Unfortunately there are no field, cell or table comments.

**Describe the solution you'd like**

Here we want to focus on comments that target specific records. When right-clicking a comment or pressing Ctrl + Shift + M, it is possible to add cell comments.

### LINKED ISSUE #835: Table comments
**Is your feature request related to a problem? Please describe.**

Data are social artifacts which often need discussion. Unfortunately there are no field, record or table comments.

**Describe the solution you'd like**

It is possible to discuss tables with comments in a sidebar.

### PR DESCRIPTION
**Pull request by caoxing9**
feat: support record comment
- [x] support link
- [x] support @ user
- [x] support image
- [x] support reaction
- [x] support notification
- [x] realtime conversation

https://github.com/user-attachments/assets/34ba21b7-a055-4e95-aaca-0921c4a58686

comment-gif

### CODE REVIEW

File: packages/db-main-prisma/prisma/template.prisma [hunk 1]
```diff
@@ -395,3 +395,29 @@ model RecordHistory {
   @@index([tableId, createdTime])
   @@map("record_history")
 }
+
+model Comment {
+  id               String     @id @default(cuid())
+  tableId          String     @map("table_id")
+  recordId         String     @map("record_id")
+  quoteId          String?    @map("quoteId")
```
Comment (tea-artist) [inline packages/db-main-prisma/prisma/template.prisma]: @map("quote_id")

File: packages/sdk/src/components/comment/comment-list/reaction/ReactionPicker.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    unified: '1f680',
+    unifiedWithoutSkinTone: '1f680',
+  },
+  {
+    key: 'tada',
+    value: `🎉`,
+    unified: '1f389',
+    unifiedWithoutSkinTone: '1f389',
+  },
+];
```
Comment (boris-w) [inline packages/sdk/src/components/comment/comment-list/reaction/ReactionPicker.tsx]: Repeat constant in openapi.

File: packages/sdk/src/components/comment/comment-list/useCommentPatchListener.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    const receiveHandler = () => {
+      const { remotePresences } = presence;
+      !isEmpty(remotePresences) && cb?.(get(remotePresences, presenceKey));
+    };
+
+    presence.on('receive', receiveHandler);
+
+    return () => {
+      presence.unsubscribe();
```
Comment (boris-w) [inline packages/sdk/src/components/comment/comment-list/useCommentPatchListener.ts]: Need ensure listenerCount === 0

File: packages/db-main-prisma/prisma/template.prisma [hunk 4]
```diff
@@ -407,3 +407,29 @@ model Trash {
...
+  recordId         String     @map("record_id")
+  quoteId          String?    @map("quote_Id")
+  content          String?
+  reaction         String?
+
+  deletedTime      DateTime?  @map("deleted_time")
+  createdTime      DateTime   @default(now()) @map("created_time")
+  createdBy        String     @map("created_by")
+  lastModifiedTime DateTime?  @updatedAt @map("last_modified_time")
+
```
Comment (Sky-FE) [inline packages/db-main-prisma/prisma/template.prisma:479]: You can add a composite index for `tableId` and `recordId`.

**Review by Sky-FE — APPROVED** [review]

File: packages/common-i18n/src/locales/ja/sdk.json [hunk 5]
```diff
@@ -206,7 +206,7 @@
       "lookup": "関連テーブルから検索",
       "button": "ボタン",
       "createdBy": "作成者",
-      "lastModifiedBy": "最終更新者"
+      "lastModifiedBy": "最終更新��"
```
Comment (tea-artist) [inline packages/common-i18n/src/locales/ja/sdk.json]: character error

File: packages/common-i18n/src/locales/ja/sdk.json [hunk 6]
```diff
@@ -611,7 +611,7 @@
     "unChecked": "チェックされていない",
     "percentEmpty": "空の割合",
     "percentFilled": "埋められた割合",
-    "percentUnique": "固有の割合",
+    "percentUnique": "固有の割��",
```
Comment (tea-artist) [inline packages/common-i18n/src/locales/ja/sdk.json]: character error

**Review by tea-artist — APPROVED** [review]

**Comment by tea-artist** [comment]
Let's rock!

**Pull request merged**